files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/tools.yml

```diff
@@ -25,6 +25,11 @@ tools:
           training_labels = '"' + ", ".join(training_roles) + '"'
           entity.params['requirements'] = '(GalaxyGroup == "compute") || (%s)' % training_expr if training_expr else '(GalaxyGroup == "compute")'
           entity.params['+Group'] = training_labels
+      - id: gpu_tools_to_condor_gpu
+        if: gpus > 0
+        scheduling:
+          require:
+            - condor-gpu
```

Comment:
(I'm not sure this fits into this PR)
I think we should think carefully about the ontology of tags that we are using here and discuss this with @cat-bro . Having something like condor in the tag name might not be portable to instances (others than EU) that do not have condor. I think I would like to see tags very generic in their ontology so that we can share as much as possible (tool def, destinations ...) with other instances.

Reply:
Sure, it probably also needs some discussions and an agreed terminology.

Reply:
We actually did some changes regarding this that are still on the VSCode server, they should show up once we commit everything.